Cargo.toml

```diff
@@ -158,27 +158,27 @@ ethers-signers = { version = "2.0.14", default-features = false }
-alloy-signer-ledger = { git = "https://github.com/alloy-rs/alloy", rev = "987b393", default-features = false }
-alloy-signer-trezor = { git = "https://github.com/alloy-rs/alloy", rev = "987b393", default-features = false }
-alloy-transport = { git = "https://github.com/alloy-rs/alloy", rev = "987b393", default-features = false }
-alloy-transport-http = { git = "https://github.com/alloy-rs/alloy", rev = "987b393", default-features = false }
-alloy-transport-ipc = { git = "https://github.com/alloy-rs/alloy", rev = "987b393", default-features = false }
-alloy-transport-ws = { git = "https://github.com/alloy-rs/alloy", rev = "987b393", default-features = false }
+alloy-consensus = { git = "https://github.com/alloy-rs/alloy", rev = "44c905d", default-features = false }
+alloy-contract = { git = "https://github.com/alloy-rs/alloy", rev = "44c905d", default-features = false }
+alloy-eips = { git = "https://github.com/alloy-rs/alloy", rev = "44c905d", default-features = false }
+alloy-genesis = { git = "https://github.com/alloy-rs/alloy", rev = "44c905d", default-features = false }
+alloy-json-rpc = { git = "https://github.com/alloy-rs/alloy", rev = "44c905d", default-features = false }
+alloy-network = { git = "https://github.com/alloy-rs/alloy", rev = "44c905d", default-features = false }
+alloy-node-bindings = { git = "https://github.com/alloy-rs/alloy", rev = "44c905d", default-features = false }
+alloy-provider = { git = "https://github.com/alloy-rs/alloy", rev = "44c905d", default-features = false }
+alloy-pubsub = { git = "https://github.com/alloy-rs/alloy", rev = "44c905d", default-features = false }
+alloy-rpc-client = { git = "https://github.com/alloy-rs/alloy", rev = "44c905d", default-features = false }
+alloy-rpc-types-trace = { git = "https://github.com/alloy-rs/alloy", rev = "44c905d", default-features = false }
+alloy-rpc-types = { git = "https://github.com/alloy-rs/alloy", rev = "44c905d", default-features = false }
+alloy-signer = { git = "https://github.com/alloy-rs/alloy", rev = "44c905d", default-features = false }
+alloy-signer-wallet = { git = "https://github.com/alloy-rs/alloy", rev = "44c905d", default-features = false }
+alloy-signer-aws = { git = "https://github.com/alloy-rs/alloy", rev = "44c905d", default-features = false }
+alloy-signer-ledger = { git = "https://github.com/alloy-rs/alloy", rev = "44c905d", default-features = false }
+alloy-signer-trezor = { git = "https://github.com/alloy-rs/alloy", rev = "44c905d", default-features = false }
+alloy-transport = { git = "https://github.com/alloy-rs/alloy", rev = "44c905d", default-features = false }
+alloy-transport-http = { git = "https://github.com/alloy-rs/alloy", rev = "44c905d", default-features = false }
+alloy-transport-ipc = { git = "https://github.com/alloy-rs/alloy", rev = "44c905d", default-features = false }
+alloy-transport-ws = { git = "https://github.com/alloy-rs/alloy", rev = "44c905d", default-features = false }
```

Comment:
Recommended to upgrade to `188c4f8` which includes a fix for the `Caused by: missing properties: [("NonceManager", ["from"]), ("Signer", ["TODO"])] ` error we encountered